solidus_braintree.gemspec

```diff
@@ -34,11 +34,11 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'solidus_api', ['>= 2.4.0', '< 4']
   spec.add_dependency 'solidus_core', ['>= 2.4.0', '< 4']
   spec.add_dependency 'solidus_support', ['>= 0.8.1', '< 1']
+  spec.add_dependency 'vcr'
+  spec.add_dependency 'webmock'
```

Comment:
If this is required in the target app as for running specs maybe it would be better to have them added to the gemfile by the installer, thoughts?

Reply:
@elia That makes sense since they're only used for the specs and are not actual dependencies of the extension. I'll try letting the installer append them to the Gemfile :+1:

Reply:
@elia I'm having some issues getting InstallGenerator to install the vcr and webmock gems in CI. See https://app.circleci.com/pipelines/github/solidusio/solidus_braintree/255/workflows/13c4ce91-4091-4965-b09e-d60a26cf9e27. I've moved back the gems to the gemspec for now. Will investigate tomorrow...